### DISCUSSION TITLE
BootstrapBlazorRoot标签的OnErrorHandleAsync，设置后没有任何效果

### DISCUSSION THREAD

**Discussion by WeiJunFenYou** [dotnetcore/BootstrapBlazor#3993] [Q&A]
`<BootstrapBlazorRoot EnableErrorLogger="true" OnErrorHandleAsync="(logger,ex)=> SwalService.Show(new SwalOption()
    {
        Category = SwalCategory.Information,
        Title = ex.Message,
    })">`

我设置了如上的自定义错误处理。但在发生异常后，页面没有产生任何错误弹窗。

![image](https://github.com/user-attachments/assets/d212517d-e748-433b-9069-25188ddf0e41)

**Comment by WeiJunFenYou** [comment]
完整的复现代码在此https://github.com/WeiJunFenYou/BootstrapBlazor.UiBugTest

**Reply by ArgoZhang** [reply]
@WeiJunFenYou 看了一下你的代码，你这 。。。。 一言难尽


建议看看视频教程   https://space.bilibili.com/660853738/search/video?keyword=全局异常
官网文档  https://www.blazor.zone/global-exception

先把基础知识点学一下

**Reply by WeiJunFenYou** [reply]
好好好，感谢作者🙏
